core/src/main/java/org/apache/iceberg/jdbc/JdbcUtil.java

```diff
@@ -303,7 +287,294 @@ public static Properties filterAndRemovePrefix(Map<String, String> properties, S
     return result;
   }
 
-  public static String updatePropertiesStatement(int size) {
+  static String getSql(boolean isTable) {
+    String tableName = isTable ? CATALOG_TABLE_NAME : CATALOG_VIEW_NAME;
+    String namespace = isTable ? TABLE_NAMESPACE : VIEW_NAMESPACE;
+    String name = isTable ? TABLE_NAME : VIEW_NAME;
+    return String.format(GET, tableName, namespace, name);
+  }
+
+  static String getTableSql() {
```

Comment:
this one still has a `get` prefix. Maybe rename this to `tableSql()` or `sqlForTable()`, but the former probably better aligns with the other method naming